Gemfile

```diff
@@ -20,6 +20,7 @@ gem "gds-sso", "12.1.0"
 gem 'bunny', '2.5.1'
 gem 'whenever', '0.9.4', require: false
 gem "govuk_sidekiq", "~> 0.0"
+gem 'sidekiq', git: "https://github.com/alphagov/sidekiq.git", branch: "fix-testing-with-overridden-queue"
```

Comment:
I think having a specific branch is clearer than using master. The fix-testing-with-overridden-queue branch is based on the v4.1.4 tag, which is the version we are currently using in the Publishing API.This is why the branch is behind master, I just wanted to add this specific patch.

I created an issue in the mperham repository https://github.com/mperham/sidekiq/issues/3144